script.js

```diff
@@ -1,52 +1,58 @@
-let bookArray = [];
-
 function Book(name, author) {
   this.name = name;
   this.author = author;
 }
+class Books {
+  constructor() {
+    this.storage = [];
+  }
 
+  add(book) {
+    this.storage.push(book);
+  }
```

Comment:
- [ ] (Optional) I think it would be better not to mutate the array of books by using a `concat` method instead of a `push` method. Your code could look like this:

```
this.storage = this.storage.concat(book);
```